### ISSUE TITLE
Cannot read property 'toLowerCase' of undefined on Vue 3 fresh installation

### ISSUE THREAD

**Issue by Filip785** [iamstevendao/vue-tel-input#277]
Hello there,

I have fresh installation of Vue 3 and I'm trying to use `vue-tel-input` in my component, but I get the following error:

> Uncaught TypeError: Cannot read property 'toLowerCase' of undefined

The error refers to [this line](https://github.com/iamstevendao/vue-tel-input/blob/eef57b7115149fa9ba4ab108b6f187c7b795f738/src/components/vue-tel-input.vue#L14).

Here is my code:

```javascript
<template>
  <vue-tel-input v-model="values.mobile" mode="international"></vue-tel-input>
</template>

<script>
import { VueTelInput } from 'vue-tel-input';
import 'vue-tel-input/dist/vue-tel-input.css';

export default {
  name: 'form-fields',
  components: { VueTelInput },
  props: [
    'values',
  ]
};
</script>
```

Issue is obviously with dropdownOptions.showFlags, so I tried disabling it like so:

```javascript
<template>
    <vue-tel-input v-model="values.mobile" :dropdownOptions="{ showFlags: false }" mode="international"></vue-tel-input>
</template>

<script>
import { VueTelInput } from 'vue-tel-input';
import 'vue-tel-input/dist/vue-tel-input.css';

export default {
  name: 'form-fields',
  components: { VueTelInput },
  props: [
    'values',
  ]
};
</script>
```

However, this gives me new error:

> Uncaught TypeError: Cannot read property 'name' of undefined

This error refers to [this line](https://github.com/iamstevendao/vue-tel-input/blob/d7ceedc9c693633fb11f4491ced7bdf619b067e8/src/directives/click-outside.js#L18).

I am using [`vue-tel-input@next`](https://github.com/iamstevendao/vue-tel-input/tree/next).

Any tips on how to solve this would be highly appreciated.
Thanks.

**Comment by krckyboy** [comment]
I'm in the same situation with "version": "6.0.0-beta.1".

**Comment by krckyboy** [comment]
I think this is a problem related to our project config, since with a clean Vue CLI project install, this library is working fine. We're using Vue for just parts of code, and in this case, each form is a Vue app. We're using Symfony (PHP) with Twig. So it's worth pointing out that this is not a SPA.

I'm creating Vue widgets like this: 
```javascript
// initializeWidget.js

const app = createApp({ render: () => h(widget, { ...dataset }) });
app.mount('#app');
```
```javascript
// webpack.config.js

.addPlugin(
  new webpack.DefinePlugin({
    __VUE_OPTIONS_API__: false,
    __VUE_PROD_DEVTOOLS__: false
  })
)

.enableVueLoader(() => {}, {
  version: 3,
  runtimeCompilerBuild: false,
})
```

Any clue what we're missing to make this work?

**Comment by iamstevendao** [comment]
Hey guys, it should be fixed in `v6.0.0-beta.3`. Feel free to re-open if you find any issues.
